rasterio/transform.py

```diff
@@ -60,13 +60,13 @@ def xy(
         col : int
             Pixel column.
         z : float, optional
-            Height associated with coordinates. Primarily used for RPC based 
-            coordinate transformations. Ignored for affine based 
```

Comment:
I looked in this module to remind myself how I was using ExitStack and found some trailing whitespace to remove. It's related :wink: